packages/flutter/lib/src/material/time_picker_theme.dart

```diff
@@ -197,7 +161,17 @@ class TimePickerThemeData with Diagnosticable {
   ///   side: BorderSide(),
   /// )
   /// ```
-  final ShapeBorder dayPeriodShape;
+  final OutlinedBorder dayPeriodShape;
+
+  /// The border side of the day period.
```

Comment:
The color and weight of the day period's outline.

Reply:
Done